content/guides/references/release-stages.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Release Stages
+---
+
+Cypress follows Semantic Versioning so that it is clear how you may be affected
+when upgrading to a newer version of Cypress. If there is a minor version
+increment, 11.0.0 → 11.1.0 for example, you should not experience disruption in
+your usage of Cypress. If there is a major version increment, 11.1.0 → 12.0.0
+for example, you may need to make changes to continue using Cypress. For most
+scenarios this is a valid way to consider complexity when upgrading Cypress. In
+some cases, however, a feature of cypress may be in a stage of development in
+which it does not adhere to these rules. In order to better provided
+transparency of when this might be the case, we have consolidated definitions of
+different release stages and how we approach changes and communication in those
+stages.
+
+## Experimental
+
+Experimental features consist of features that we are looking to gather user
+feedback on before we commit fully to the initial implementation or the feature
+entirely. These type of features require you to opt-in to via a configuration
+value before the feature is activated. These types of features may be unstable
+and contain issues. You should expect to see frequent changes and improvements
+during the feature’s experimental lifetime.
+
+**Strategies for releasing and communicating changes to experimental features**
+
+- Modifications
+  - Modifications can be introduced during any release cycle and may or may not
+    increment semver.
+- Breaking Changes
+  - Breaking changes can be introduced during any release cycle and do not bump
+    semver by a major version.
+- Removals
+  - Removal of any new functionality related to the feature or the feature in
+    its entirety can happen during any release cycle and will not be accompanied
+    by a major semver version change.
+
+## Alpha
+
+Alpha features consist of features that we are looking to gather user feedback
+on by providing them as default functionality before an official release. Alpha
+features may not be feature complete and changes are possible during this stage.
+
+**Strategies for releasing and communicating changes to Alpha features**
+
+- Modifications
+  - Modifications can be introduced during any release cycle and may or may not
+    increment semver.
+  - Affected users will be pointed to documentation if there blockers are
+    perceived to be possible.
+- Breaking Changes
+  - Breaking changes can be introduced during any release cycle and do not bump
+    semver by a major version
+- Removals
+  - Removal of any new functionality related to the feature can happen during
+    any release cycle and does not bump semver by a major version.
+
+## Beta
```

Comment:
We should clarify how a user can know if a feature is alpha/beta/GA etc. For CT it was easy, there was a label, but for thing like cy.origin, this might not always be the case. Any thoughts on this? I wonder if the experiments page could be expanded to have a "Stage" for each experiment - but I suppose it wouldn't really be an experiments page then, but a "pre-release features" page.